util/stringutil/string_util_test.go

```diff
@@ -65,20 +54,19 @@ func (s *testStringUtilSuite) TestUnquote(c *C) {
 		{"\"\\a\x18èàø»\x05\"", "a\x18èàø»\x05", true},
 	}
 
-	for _, t := range table {
-		x, err := Unquote(t.str)
-		c.Assert(x, Equals, t.expect)
-		comment := Commentf("source %v", t.str)
-		if t.ok {
-			c.Assert(err, IsNil, comment)
+	for _, v := range table {
+		x, err := Unquote(v.str)
+		require.Equal(t, v.expect, x)
+		if v.ok {
+			require.Nil(t, err)
```

Comment:
There is `Nilf`, and `NotNilf`, the alternative for `Commentf("source %v", t.str)`, ditto for other comments. Comments are useful for debugging.

Reply:
ok thanks, i will ehance it

Reply:
hi, i finished the enhance, please review again.
i searched whole code repo and it seems that i am the first one to use `Nilf` and `NotNilf` function.
i manually made some test fail, seems the testfiy output is clean enough and no `Nilf` related comment is show on the idea terminal. not sure this is expected or not.
```shell
/usr/local/go/bin/go tool test2json -t /tmp/___TestUnquote_in_github_com_pingcap_tidb_util_stringutil.test -test.v -test.paniconexit0 -test.run ^\QTestUnquote\E$
=== RUN   TestUnquote
=== PAUSE TestUnquote
=== CONT  TestUnquote
    string_util_test.go:59: 
        	Error Trace:	string_util_test.go:59
        	Error:      	Not equal: 
        	            	expected: "abcdef"
        	            	actual  : "123abcdef"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-abcdef
        	            	+123abcdef
        	Test:       	TestUnquote
--- FAIL: TestUnquote (0.00s)
```

Reply:
I'd suggest you write "source %v" to cover the case where v=""

Reply:
get it